### PR TITLE
esdt improvements integration

### PR DESCRIPTION
**Pull request by BeniaminDrasovean**
## Reasoning behind the pull request
- 
- 
- 
  
## Proposed changes
- 
- 
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: common/enablers/enableEpochsHandler.go [hunk 1]
```diff
@@ -697,9 +697,9 @@ func (handler *enableEpochsHandler) createAllFlagsMap() {
 		},
 		common.DynamicESDTFlag: {
 			isActiveInEpoch: func(epoch uint32) bool {
-				return epoch >= handler.enableEpochsConfig.NFTStopCreateEnableEpoch
+				return epoch >= handler.enableEpochsConfig.DynamicESDTEnableEpoch
```
Comment (iulianpascalau) [inline common/enablers/enableEpochsHandler.go:700]: unit test?

File: factory/disabled/globalSettingsHandler.go [hunk 2]
```diff
@@ -7,6 +7,26 @@ func NewDisabledGlobalSettingHandler() *disabledGlobalSettingsHandler {
...
+	return false
+}
+
+// GetTokenType returns 0 as this is a disabled component
+func (d *disabledGlobalSettingsHandler) GetTokenType(_ []byte) (uint32, error) {
+	return 0, nil
+}
+
+// SetTokenType does nothing as this is a disabled component
+func (d *disabledGlobalSettingsHandler) SetTokenType(_ []byte, _ uint32) error {
```
Comment (iulianpascalau) [inline factory/disabled/globalSettingsHandler.go:26]: this is no longer a GlobalSettingHandler. Why do we have this function here?

File: common/enablers/enableEpochsHandler.go (hunk 1 above)
Comment (BeniaminDrasovean) [reply common/enablers/enableEpochsHandler.go:712]: Added

File: integrationTests/vm/txsFee/esdtMetaDataRecreate_test.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/integrationTests/vm"
+	"github.com/multiversx/mx-chain-go/integrationTests/vm/txsFee/utils"
+	vmcommon "github.com/multiversx/mx-chain-vm-common-go"
+	"github.com/stretchr/testify/require"
+)
+
+func TestESDTMetaDataRecreate(t *testing.T) {
+	tokenTypes := getDynamicTokenTypes()
+	for _, tokenType := range tokenTypes {
```
Comment (iulianpascalau) [inline integrationTests/vm/txsFee/esdtMetaDataRecreate_test.go:20]: :+1:

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**